constraints/BUILD

```diff
@@ -24,19 +24,14 @@ constraint_value(
     constraint_setting = ":linux_distribution",
 )
 
-# RBE Debian8
-platform(
-    name = "rbe_debian8",
-    constraint_values = [
-        "@bazel_tools//platforms:x86_64",
-        "@bazel_tools//platforms:linux",
-        "@bazel_tools//tools/cpp:clang",
-        ":jessie",
-    ],
-    remote_execution_properties = """
-          properties: {
-            name: "container-image"
-            value:"docker://gcr.io/cloud-marketplace/google/rbe-debian8@sha256:d84a7de5175a22505209f56b02f1da20ccec64880da09ee38eaef3670fbd2a56"
-         }
-         """,
+constraint_value(
+    name = "xenial",
+    constraint_setting = ":linux_distribution",
+)
+
+constraint_setting(name = "msan")
```

Comment:
name = sanitizer

Reply:
or sanitizer_support

Reply:
I tried that. I had this before

constraint_setting(name = "sanitizer")

constraint_value(
    name = "no_sanitizer",
    constraint_setting = ":sanitizer",
)

constraint_value(
    name = "msan",
    constraint_setting = ":sanitizer",
)

But it doesn't work, for one platform, it can only have one value for one constraint_settings. So if a platform supports msan, tsan, no_sanitizer, I couldn't do it unless create duplicate platforms, each has one sanitizer constraint value, and with the same container sha.

We sort of have to do it in the support_msan, support_tsan, way.

Reply:
got it, lgtm as it is then

Reply:
Oh, I think there's a better way though. In constraints create folder sanitizers with a build file there, call the constraint just msan